test-clients/src/main/java/com/hedera/services/bdd/suites/contract/OZContracts/OZContract_ERC1155.java

```diff
@@ -0,0 +1,163 @@
+	private static final String ABI_SAFE_BATCH_TRANSFER_FROM = "{\"inputs\":[{\"internalType\":\"address\",\"name\":\"from\",\"type\":\"address\"},{\"internalType\":\"address\",\"name\":\"to\",\"type\":\"address\"},{\"internalType\":\"uint256[]\",\"name\":\"ids\",\"type\":\"uint256[]\"},{\"internalType\":\"uint256[]\",\"name\":\"amounts\",\"type\":\"uint256[]\"},{\"internalType\":\"bytes\",\"name\":\"data\",\"type\":\"bytes\"}],\"name\":\"safeBatchTransferFrom\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"}";
+	private static final String ABI_BALANCE_OF = "{\"inputs\":[{\"internalType\":\"address\",\"name\":\"account\",\"type\":\"address\"},{\"internalType\":\"uint256\",\"name\":\"id\",\"type\":\"uint256\"}],\"name\":\"balanceOf\",\"outputs\":[{\"internalType\":\"uint256\",\"name\":\"\",\"type\":\"uint256\"}],\"stateMutability\":\"view\",\"type\":\"function\"}";
+	private static final String ABI_APPROVE = "{\"inputs\":[{\"internalType\":\"address\",\"name\":\"operator\",\"type\":\"address\"},{\"internalType\":\"bool\",\"name\":\"approved\",\"type\":\"bool\"}],\"name\":\"setApprovalForAll\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"}";
+	private static final String ABI_MINT = "{\"inputs\":[{\"internalType\":\"uint256\",\"name\":\"tokenType\",\"type\":\"uint256\"},{\"internalType\":\"uint256\",\"name\":\"amount\",\"type\":\"uint256\"},{\"internalType\":\"address\",\"name\":\"recipient\",\"type\":\"address\"}],\"name\":\"mintToken\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"}";
+	
+	private static final String CONTRACT_NAME = "ERC-1155";
+	private static final String OPERATIONS_PAYER = "payer";
+	private static final String ACCOUNT1 = "acc1";
+	private static final String ACCOUNT2 = "acc2";
+
+	public static void main(String... args) {
+		new OZContract_ERC1155().runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				erc1155()
+		);
+	}
+
+	private HapiApiSpec erc1155() {
+		ByteString contents = null;
+		try {
+			var bytes = Files.readAllBytes(Path.of(ContractResources.ERC_1155_BYTECODE_PATH));
+			contents = ByteString.copyFrom(bytes);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		return defaultHapiSpec(CONTRACT_NAME)
+				.given(
+						cryptoCreate(ACCOUNT1),
+						cryptoCreate(ACCOUNT2),
+						cryptoCreate(OPERATIONS_PAYER).balance(ONE_MILLION_HBARS),
+						fileUpdate(APP_PROPERTIES)
+								.payingWith(ADDRESS_BOOK_CONTROL)
+								.overridingProps(Map.of(
+										"contracts.maxStorageKb", "" + Integer.MAX_VALUE,
+										"contracts.maxGas", "" + Integer.MAX_VALUE)),
+						fileCreate("bytecode").contents("").payingWith(OPERATIONS_PAYER),
```

Comment:
You do not need the empty contents statement. Also "bytecode" deceives the reader **a lot**. Can we rename that?